docs/software.md

```diff
@@ -2,12 +2,13 @@
 
 ## Overview
 
-**DietPi-Software** is a core DietPi tool, allowing you to install **Ready to run** and **Optimized software** for your device. It automates the installation and configuration process for you and also applies specific optimizations unique to your hardware model. As a result DietPi-Software ensure you get the **maximum performance** from your device.
+**DietPi-Software** is a core DietPi tool, allowing you to install **Ready to run** and **Optimized software items** for your device. It automates the installation and configuration process for you and also applies specific optimizations unique to your hardware model. As a result DietPi-Software ensure you get the **maximum performance** from your device.
 
-??? info "How do I run **DietPi-Software** and install **Optimised software** ?"
-    To install any of the **DietPi Optimised Software** listed below run from the command line `dietpi-launcher` and selected *DietPi-Software* or launch the tool `dietpi-software`.
+??? info "How do I run **DietPi-Software** and install **Optimised software** items?"
+    To install any of the **DietPi optimised software items** listed below run from the command line `dietpi-launcher` and selected *DietPi-Software* or launch the tool `dietpi-software`.
```

Comment:
`run dietpi-launcher from the command line`.
`selected` -> `select`.

Reply:
Thx. Changed.